Sources/Swift/Integrations/UserFeedback/Configuration/SentryUserFeedbackConfiguration.swift

```diff
@@ -37,7 +37,14 @@ public class SentryUserFeedbackConfiguration: NSObject {
      * - note: Setting this to true does not disable the widget. In order to do so, you must set `SentryUserFeedbackWidgetConfiguration.autoInject` to `false` using the `SentryUserFeedbackConfiguration.configureWidget` config builder.
      */
     public var showFormForScreenshots: Bool = false
-    
+
+    /**
+     * Install a hook for the specified button to show the form when it is pressed.
+     * - note: If this is set, `configureWidget` is ignored.
+     * - note: Default: `nil`
+     */
+    public var customButton: UIButton?
+
```

Comment:
`l`: Using a button seems to conver most use-cases. To widen the possibilities without impacting the SDK we could also go for [`UIControl`](https://developer.apple.com/documentation/uikit/uicontrol) instead, allowing [other components](https://developer.apple.com/documentation/uikit/uicontrol#inherited-by) such as [UISwitch](https://developer.apple.com/documentation/uikit/uiswitch) or custom controls to be used as well.

This might be an edge case for most users, but as we are not using any `UIButton`-specific features, I don't see a reason to be restrictive either.

```suggestion
    public var customButton: UIControl?
```

Reply:
This is out of scope for this PR, please file an issue and/or discuss this idea in the project's slack room.

Reply:
Would not have been any changes other than changing the type `UIButton` to `UIControl`. I'll raise it in Slack to spread awareness, rather than requesting it.

Reply:
We only attach a touch-up-inside action trigger to the UI element. if we instead use a UIControl, then we have to consider other actions like value changed for UISwitch, so a straight swap is not possible without surprising side effects for end users. This would expand the design scope of the feature as we have to consider what it really means for people to use a switch versus a button. They are different entities in the UI framework because they are different UX. So I disagree that they can be treated interchangeably. If you feel strongly that we should support something like a switch (which really makes zero sense to me here) then please raise the issue with our design team and PMs, not on a PR like this.